ompi/request/request.h

```diff
@@ -129,7 +134,7 @@ struct ompi_request_t {
     ompi_status_public_t req_status;            /**< Completion status */
     volatile void *req_complete;                /**< Flag indicating wether request has completed */
     volatile ompi_request_state_t req_state;    /**< enum indicate state of the request */
-    bool req_persistent;                        /**< flag indicating if the this is a persistent request */
+    bool req_flags;                             /**< flag indicating properties of the request (persistency, release resources) */
```

Comment:
This should be at least two bits (for example `uint8_t`), right?

Other options to  kind-of maintain ABI compatibility is to hijack the LSB of `req_complete_cb_data` (or MSB or `req_f_to_c_index`)

Reply:
I meant to be an int as it will fit perfectly in the ompi_request_t struct. I even put it in the commit log ;)